### DISCUSSION TITLE
Dockerization

### DISCUSSION THREAD

**Discussion by madeofpendletonwool** [ChocolateApp/Chocolate#16] [General]
Hey there, I'm from reddit! I mentioned on there that I would take a crack at the docerization process here. I haven't dug in too much but I have been able to get the existing dockerfile running. Here's what I've got. 

```
FROM python:3.10-slim-buster

RUN apt-get update && \
    apt-get install -y ffmpeg && \
    apt-get install -y git && \
    apt-get install -y software-properties-common && \
    apt-add-repository non-free && \
    apt-get update && \
    apt-get install -y unrar

COPY requirements.txt .
RUN pip install -r requirements.txt

WORKDIR /chocolate
COPY . .

CMD ["python", "app.py"]
```
This builds and runs, and I've exposed port 8888 which is what flask seems to run, however, I just end up with a screen at the site that says 

"Chocolate Server"

Can I get some more information what's perhaps missing? I just don't have a base understanding of the app yet so I'm looking for some more information on how it functions in order for me to fully dockerize it. 

Hope we can work together on this!

Thanks!

**Comment by Impre-visible** [comment]
Thanks, since the new version, we need to use [Chocolate React](https://github.com/ChocolateApp/ChocolateReact). It's the new front end. Thanks for the new start of Dockerfile. I don't know how I will get the Dockerfile to download the react service. I'm gonna try to use a git clone. Thanks again
